src/definitions/subscriptionField.ts

```diff
@@ -26,7 +26,7 @@ export interface SubscribeFieldConfig<
     args: ArgsValue<TypeName, FieldName>,
     ctx: GetGen<"context">,
     info: GraphQLResolveInfo
-  ): AsyncIterator<T>;
+  ): AsyncIterator<T> | Promise<AsyncIterator<T>>;
```

Comment:
Should we use
```suggestion
  ): MaybePromise<AsyncIterator<T>> | MaybePromiseDeep<AsyncIterator<T>>;
```
here like we do for FieldResolver?

Reply:
Seems to work fine.